### PR TITLE
Feature/selenium 3 compatibility

### PR DESCRIPTION
**Pull request by elvetemedve**

### CODE REVIEW

File: lib/hem/tasks/selenium.rb [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
   desc 'Start Selenium'
   task :start do
+    jarPath = Hem::Selenium::Jar::getLocalPath();
     cmd  = [
       'export NSPR_LOG_MODULES=all:3;',
-      "java -jar './bin/selenium-server-standalone.jar'",
+      'java ',
         '-Dwebdriver.firefox.logfile=/tmp/selenium-firefox.log',
         '-Dwebdriver.chrome.driver=./bin/chromedriver',
-        '-trustAllSSLCertificates',
```
Comment (kierenevans) [inline lib/hem/tasks/selenium.rb:15]: Is this still needed for self signed certs?

Comment (elvetemedve) [reply lib/hem/tasks/selenium.rb:15]: Selenium 3.0.1 complains that there is no such option. That's why I removed it.
There must be an other way to skip certificate verification.

Comment (elvetemedve) [reply lib/hem/tasks/selenium.rb:15]: @kierenevans I did research in this topic and I found that there is a browser level setting (capability) which can turn off the verification, see https://github.com/Behat/MinkExtension/blob/master/src/Behat/MinkExtension/ServiceContainer/Driver/Selenium2Factory.php#L113
So in behat.yml should look like

``` yaml
extensions:
    Behat\MinkExtension\ServiceContainer\MinkExtension:
      sessions:
        firefox:
          selenium2:
            wd_host: http://localhost:4444/wd/hub
            browser: firefox
            acceptSslCerts: true
```

Therefore I would not bother with turning it off on a global level.

**Comment by kierenevans** [comment]
Sorry, this now has a conflict (should have merged this first!)

**Comment by elvetemedve** [comment]
@kierenevans I rebased the branch, so the merge conflict has gone away.

**Pull request merged**